src/steamship/invocable/invocable.py

```diff
@@ -180,6 +189,55 @@ def config_cls(self):
         """
         return Config
 
+    def invoke_later(
+        self,
+        method: str,
+        verb: Verb = Verb.POST,
+        wait_on_tasks: List[Union[Task, str]] = None,
+        arguments: Dict[str, Any] = None,
+    ) -> Task[Any]:
+        """Schedule a method for future invocation.
+
+        :param method: The method to invoke, as registered with Steamship in the @get or @post decorator.
```

Comment:
I guess your Pycharm is not set to use numpy docstring's.

Reply:
It is (just checked). Is that supposed to throw an error on a docstring like this?